staging/src/k8s.io/apiserver/pkg/admission/interfaces.go

```diff
@@ -49,6 +49,12 @@ type Attributes interface {
 	GetKind() schema.GroupVersionKind
 	// GetUserInfo is information about the requesting user
 	GetUserInfo() user.Info
+	// GetAnnotations returns annotation map for the admission chain.
+	GetAnnotations() map[string]string
```

Comment:
can this be queried by plugins in the chain? I don't like to open up this channel without good reasons.